.github/workflows/issue-label-assign.yml

```diff
@@ -0,0 +1,49 @@
+# ~~ Generated by projen. To modify, edit .projenrc.ts and run "npx projen".
+
+name: issue-label-assign
+on:
+  pull_request_target:
+    types:
+      - opened
+  issues:
+    types:
+      - opened
+      - edited
+jobs:
+  Triage Issues:
+    name: Triage Manager
+    runs-on: aws-cdk_ubuntu-latest_4-core
+    permissions:
+      issues: write
+      pull-requests: write
+    env:
+      AREA_PARAMS: '[{"area":"@aws-cdk/cli-lib-alpha","keywords":["cli","cli-lib","cli-lib-alpha"],"labels":["@aws-cdk/cli-lib-alpha"]},{"area":"@aws-cdk/cloud-assembly-schema","keywords":["cloud-assembly","schema"],"labels":["@aws-cdk/cloud-assembly-schema"]},{"area":"@aws-cdk/cloudformation-diff","keywords":["diff","cloudformation"],"labels":["@aws-cdk/cloudformation-diff"]},{"area":"@aws-cdk/toolkit-lib","keywords":["toolkit","programmtic toolkit","toolkit-lib"],"labels":["@aws-cdk/toolkit-lib"]},{"area":"aws-cdk","keywords":["aws-cdk","cli","cdk cli"],"labels":["aws-cdk"]},{"area":"cdk-assets","keywords":["assets","cdk-assets"],"labels":["cdk-assets"]}]'
+      AREA_AFFIXES: '{"prefixes":["@aws-cdk/"]}'
+      OSDS_DEVS: '{"assignees":["ashishdhingra","khushail","hunhsieh"]}'
+    steps:
+      - name: Triage Manager
+        uses: aws-github-ops/aws-issue-triage-manager@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          target: issues
+          excluded-expressions: "[CDK CLI Version|TypeScript|Java|Python|Go]"
+          included-labels: "[needs-triage]"
+          excluded-labels: "[p1|p2|p0|effort-small|effort-medium|effort-large|guidance]"
+          default-area: ${{ env.OSDS_DEVS }}
+          parameters: ${{ env.AREA_PARAMS }}
+          affixes: ${{ env.AREA_AFFIXES }}
+  Triage Pull Requests:
+    name: Triage Manager
+    runs-on: aws-cdk_ubuntu-latest_4-core
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Triage Manager
+        uses: aws-github-ops/aws-issue-triage-manager@main
+        with:
+          github-token: ${{ secrets.PROJEN_GITHUB_TOKEN }}
```

Comment:
https://github.com/aws/aws-cdk/pull/19727/files this doesn't tell me why but it seems it was intentional